Exadel.Compreface.UnitTests/ApiClient/ApiClientTests.HttpVerbs.cs

```diff
@@ -0,0 +1,112 @@
+using Exadel.Compreface.DTOs.ExampleSubjectDTOs.AddExampleSubject;
+using Exadel.Compreface.DTOs.SubjectDTOs.GetSubjectList;
+using Moq;
+
+namespace Exadel.Compreface.UnitTests.ApiClient;
+
+public partial class ApiClientTests
+{
+    [Fact]
+    public async Task GetJsonAsync_ShouldUseGETHttpMethod()
+    {
+        // simulating api calls here and constructing fake response
+        _httpTest.RespondWith(SuccessMessage, 200);
```

Comment:
Changed 400 to BadRequestStatusCode, but 200 is not 😄

Reply:
Yeah, I forgot to do it